src/firefly/js/ui/tap/TemportalSearch.jsx

```diff
@@ -129,8 +130,10 @@ export function TemporalSearch({cols, columnsModel}) {
     useEffect(() => {
         const timeCol= findTimeColumn(columnsModel) ?? '';
         const errMsg= 'Temporal searches require identifying a table column containing a time in MJD.  Please provide a column name.';
-        setVal(TemporalColumns, timeCol, {validator: getColValidator(cols, true, false, errMsg), valid: true});
-        checkHeaderCtl.setPanelOpen(Boolean(timeCol));
+        //if (timeCol != '') setVal(TemporalColumns, timeCol, {validator: getColValidator(cols, true, false, errMsg), valid: true});
+        //setVal(TemporalColumns, timeCol, {validator: getColValidator(cols, true, false, errMsg), valid: true});
+        if (Boolean(timeCol)) checkHeaderCtl.setPanelOpen(true);
+        //checkHeaderCtl.setPanelOpen(Boolean(timeCol));
```

Comment:
It's my understanding that we were setting the temporal panel to open only if the columnsModel had a time column (as returned by the **findTimeColumn** function) and also setting the temporal column value to this timeCol value returned. That's definitely good UX when timeCol exists but it leads to a few issues when it does not. 

This resulted in the panel automatically closing if you open it, leave the TAP panel (say you go to Edit ADQL) and come back to UI Assisted mode and the Temporal search panel is closed again. That's fixed by the if statement on line 135 here. 

The second issue here is that we were trying to setVal of the **TemporalColumns**  to **timeCol**, which is why even if you selected a temporal column value, and navigated away (to **Edit ADQL**) and back to **UI Assisted**, you would lose your selected column because **timeCol** may be empty.

A potential solution for this is on line 133, we check if a **timeCol** value exists, and only then set the temporal column's value to it. But an issue I find with this is that if the user selects a _different_ temporal column value, and they navigate away from the page and back to it, their selected value will be replaced by the default **timeCol** value returned by **findTimeColumn**. I suggest just leaving this blank and letting the user pick the column, even if one may exist in the columnsModel?

Reply:
I am not quite following. I need to understand the problem. We should talk about the issue. 
> I suggest just leaving the temporal column value blank and letting the user pick the column, even if one may exist in the columnsModel?

Rubin specifically asked us to set the column if it exist.

Reply:
Yes we can talk about this, I can share my screen as well when we do.